File: src/Reference.CSharp/Migrations/GeoContextModelSnapshot.cs
using Microsoft.EntityFrameworkCore;
using Microsoft.EntityFrameworkCore.Infrastructure;
using Microsoft.EntityFrameworkCore.Metadata;
using System;

namespace Reference.CSharp.Migrations
{
    /// <summary>
    /// Geo Context Model Snapshot
    /// </summary>
    [DbContext(typeof(GeoContext))]
    public class GeoContextModelSnapshot : ModelSnapshot
    {
        protected override void BuildModel(ModelBuilder modelBuilder)
        {
#pragma warning disable 612, 618
            modelBuilder
                .HasAnnotation("ProductVersion", "2.2.3-servicing-35854")
                .HasAnnotation("Relational:MaxIdentifierLength", 128)
                .HasAnnotation("SqlServer:ValueGenerationStrategy", SqlServerValueGenerationStrategy.IdentityColumn);

            modelBuilder.Entity("Reference.GeoData", b =>
            {
                b.Property<Guid>("Uuid")
                    .ValueGeneratedOnAdd()
                    .HasAnnotation("SqlServer:ValueGenerationStrategy", SqlServerValueGenerationStrategy.IdentityColumn);
                b.Property<double>("Latitude");
                b.Property<double>("Longitude");
                b.Property<long>("PlaceId");
                b.Property<string>("DisplayName");
                b.Property<string>("City");
                b.Property<string>("County");
                b.Property<string>("State");
                b.Property<string>("Country");
                b.Property<string>("CountryCode");

                b.HasKey("Uuid");

                b.ToTable("GeoData");
            });

#pragma warning restore 612, 618
        }
    }
}

File: src/Reference.VB/Migrations/GeoContextModelSnapshot.vb
Imports Microsoft.EntityFrameworkCore
Imports Microsoft.EntityFrameworkCore.Infrastructure
Imports Microsoft.EntityFrameworkCore.Metadata
Imports Microsoft.EntityFrameworkCore.Metadata.Builders
Imports System

Namespace Migrations

    ''' <summary>
    ''' Geo Context Model Snapshot
    ''' </summary>
    <DbContext(GetType(GeoContext))>
    Public Class GeoContextModelSnapshot
        Inherits ModelSnapshot

        Protected Overrides Sub BuildModel(modelBuilder As ModelBuilder)

            modelBuilder _
            .HasAnnotation("ProductVersion", "2.2.3-servicing-35854") _
            .HasAnnotation("Relational:MaxIdentifierLength", 128) _
            .HasAnnotation("SqlServer:ValueGenerationStrategy", SqlServerValueGenerationStrategy.IdentityColumn)

            modelBuilder.Entity("Reference.GeoData",
                                Sub(entity As EntityTypeBuilder)
                                    entity.Property(Of Guid)("Uuid").ValueGeneratedOnAdd().HasAnnotation("SqlServer:ValueGenerationStrategy", SqlServerValueGenerationStrategy.IdentityColumn)
                                    entity.Property(Of Double)("Latitude")
                                    entity.Property(Of Double)("Longitude")
                                    entity.Property(Of Long)("PlaceId")
                                    entity.Property(Of String)("DisplayName")
                                    entity.Property(Of String)("City")
                                    entity.Property(Of String)("County")
                                    entity.Property(Of String)("State")
                                    entity.Property(Of String)("Country")
                                    entity.Property(Of String)("CountryCode")

                                    entity.HasKey("Uuid")

                                    entity.ToTable("GeoData")
                                End Sub)

        End Sub

    End Class

End Namespace
